src/python/bot/tasks/progression_task.py

```diff
@@ -210,6 +213,36 @@ def _save_fixed_range(testcase_id, min_revision, max_revision):
   task_creation.request_bisection(testcase, 'fixed')
 
 
+def _store_testcase_for_regression_testing(testcase, testcase_file_path):
```

Comment:
Are regressions made public right after they are fixed? If not how can we use them in CIFuzz?

Reply:
This is not complete, i am thinking to do hooks into corpus backup cron that makes things public. Still thinking if these regressions should be part of corpus backup or its own archive/folder.

Reply:
IMO backup is better, as that way we'll have a single good regression test suite for each fuzz target.

Reply:
Yes i agree, @jonathanmetzman - wdyt ?

Reply:
This is now done and working - https://storage.cloud.google.com/clusterfuzz-libfuzzer-backup/corpus/libfuzzer/grpc_api_fuzzer/latest.zip (see regressions sub-dir). @jonathanmetzman fyi!